caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
So according to the email that you forwarded to me from ITS, it sounds like there is no guarantee that "@uchicago.edu" iff Google login. (Correct me if I'm wrong here). 

So we would need to do some work on our end. Is it possible to force all "@uchicago.edu" people to go to app engine. If that fails, redirect back to the app and then whitelist their "@uchicago.edu" email for form field?